README.md

```diff
@@ -35,6 +35,7 @@ as its underlying SMT solver. Advanced users may wish to install alternative sol
 [Yices 2](http://yices.csl.sri.com/index.shtml),
 [CVC4](http://cvc4.cs.nyu.edu/web/), or
 [MathSAT](http://mathsat.fbk.eu/).
+[CVC5](https://github.com/cvc5/cvc5/releases/).
```

Comment:
Fix English construction.  Probably should read as "... install alternative solvers Z3, Yices (version 1), Yices 2, CVC4, CVC5, or MathSAT."

Reply:
Resolved by commit [e8acdf6](https://github.com/loonwerks/jkind/pull/79/commits/e8acdf66c25ec4b4da8ecb6bdb4b722bf32f86c4)